.github/workflows/test.yaml

```diff
@@ -0,0 +1,87 @@
+name: Test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  test-skaffold:
+    name: Skaffold
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Skaffold
+        run: curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64 && sudo install skaffold /usr/local/bin/
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Start Minikube
+        run: minikube start --uuid 00000000-0000-0000-0000-000000000001 --extra-config=apiserver.enable-admission-plugins=ValidatingAdmissionWebhook
+
+      - name: Deploy Admission Controller
+        run: ./scripts/deploy.sh
+
+      - name: Preview namespace state
+        run: kubectl -n monokle-admission-controller get all,CustomResourceDefinition,ValidatingWebhookConfiguration,secrets
+
+      - name: Install Dependencies
+        working-directory: ./tests/
+        run: npm ci
+
+      - name: Test
+        working-directory: ./tests/
+        run: npm test
+
+  test-helm:
+    name: Helm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Start Minikube
+        run: minikube start --uuid 00000000-0000-0000-0000-000000000001 --extra-config=apiserver.enable-admission-plugins=ValidatingAdmissionWebhook
+
+      - name: Build local images (init container)
+        run: eval $(minikube -p minikube docker-env) && minikube image build -t admission-webhook-init -f ./Dockerfile .
+        working-directory: ./admission-controller/init
+
+      - name: Build local images (server container)
+        run: eval $(minikube -p minikube docker-env) && minikube image build -t admission-webhook -f ./Dockerfile .
+        working-directory: ./admission-controller/server
+
+      - name: Helm install
+        run: |
+          eval $(minikube -p minikube docker-env)
+          helm package ./helm --version 0.0.0
+          helm install monokle-ac monokle-admission-controller-0.0.0.tgz \
+            --set namespace=mac-test \
+            --set image.init.pullPolicy=Never \
+            --set image.init.overridePath=admission-webhook-init \
+            --set image.server.pullPolicy=Never \
+            --set image.server.overridePath=admission-webhook
+
+      - name: Preview namespace state
+        run: kubectl -n mac-test get all,CustomResourceDefinition,ValidatingWebhookConfiguration,secrets
```

Comment:
Similarly to the above, I'd say we test more than one Helm version.
Not necesarily to be done now tho, yet I would personally do it sooner rather than later as other work will prevent us from touching this until there will be problems already.

Reply:
See my https://github.com/kubeshop/monokle-admission-controller/pull/8#issuecomment-1770716430.